src/wdio/services/WebpackDevServerService.js

```diff
@@ -25,6 +25,8 @@ export default class WebDevServerService {
   // eslint-disable-next-line class-methods-use-this
   startWebpackDevServer() {
     return new Promise((resolve, reject) => {
+      // eslint-disable-next-line no-console
+      console.error('[WebDevService] WARNING: ExpressDevServer service should be used instead of the Terra provided WebpackDevServerService. See https://github.com/cerner/terra-toolkit/blob/master/docs/ExpressDevServerService.md');
```

Comment:
Does throwing an error make prevent the service from starting?

Reply:
Potentially, I'd say we change this `console.warn`.

Reply:
I based the decision for error based on this already added depreciation warning. https://github.com/cerner/terra-toolkit/blob/master/src/nightwatch/nightwatch.config.js#L14
Do we want to change that to a `warn` too?

Reply:
I think one might actually be an alias of the other: https://nodejs.org/api/console.html#console_console_warn_data_args